engine/engine.py

```diff
@@ -87,7 +87,34 @@ class NewEngineNodeTree(bpy.types.Operator):
     bl_label = "New Node Tree"
 
     def execute(self, context):
-        bpy.ops.node.new_node_tree(type="DreamTexturesNodeTree")
+        # TODO: Get the name of the default node tree from a config file
+        # TODO: type is deprecated https://docs.blender.org/api/current/bpy.types.NodeTree.html 
+        #       When testing the type of the resulting node tree using  bpy.data.node_groups['Dream Textures Node Tree'].type it is '' because of that
+        bpy.ops.node.new_node_tree(type="DreamTexturesNodeTree", name="Dream Textures Node Editor")
+        # Get the newly generated node tree
+        node_trees = []
+        for nt in bpy.data.node_groups:
+            # the correct identifier is bl_idname nowadays
+            if nt.bl_idname == 'DreamTexturesNodeTree' and 'Dream Textures Node Editor' in nt.name:
```

Comment:
Immediately after constructing the node tree, it should be the last group in `bpy.data.node_groups`. Could it be accessed with `bpy.data.node_groups[-1]` instead of searching by name?

This is done on L102, but only after filtering by name/type.

Reply:
It works and simplifies the code quiet a bit, thanks 👍🏻